integrations/optimum/src/haystack_integrations/components/embedders/optimum/quantization.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict
+
+from optimum.onnxruntime.configuration import AutoQuantizationConfig, QuantizationConfig
+
+
+class OptimumEmbedderQuantizationMode(Enum):
+    """
+    [Dynamic Quantization Modes](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/quantization)
+    support by the Optimum Embedders.
+    """
+
+    #: Quantization for the ARM64 architecture.
+    ARM64 = "arm64"
+
+    #: Quantization with AVX-2 instructions.
+    AVX2 = "avx2"
+
+    #: Quantization with AVX-2 instructions.
```

Comment:
I think you meant `AVX-512 instructions` here

Reply:
Fixed!